index.js

```diff
@@ -17,219 +24,240 @@ const TOKEN_PATH = 'token.json';
+const CHANNEL_ID = (testing === true ? '864768873270345788' : '751893730117812225');
 const SPREADSHEET_ID = (testing === true ? '1-xVpzfIVr76dSuJO8SO-Im55WQZd0F07IQNt-hhu_po' : '1qQBxqku14GTL70o7rpLEQXil1ghXEHff7Qolhu0XrMs');
 
-
-const START_TIME = (skipstat === true ? 0 : 5)
-const END_TIME = (skipstat === true ? 24 : 21)
+const START_TIME = (skipstat === true ? 0 : 5);
+const END_TIME = (skipstat === true ? 24 : 21);
 
 const now = new Date();
 
-const isBotEnabled = botState => {
-  let nowHour = new Date().getHours();
+const isBotEnabled = (botState) => {
+  const nowHour = new Date().getHours();
   return botState === 'GO' && START_TIME < nowHour && nowHour < END_TIME;
-}
-
-
-const formatMatchRow = row => '\u200b' + row[0] + '\u200b' + ((typeof row[1] != 'undefined') ? ` ${row[1]}` : '');
+};
 
+const formatMatchRow = (row) => `\u200b${row[0].trim()}\u200b${(typeof row[1] !== 'undefined') ? ` ${row[1]}` : ''}`;
+const formatOther = (text) => text.replaceAll(/:[^:\n]+:/g, '\u200b$1\u200b');
 
-const getMatchText = rows => rows.map(formatMatchRow).join('\n');
+const getMatchText = (rows) => rows.map(formatMatchRow).join('\n');
 
-
-const getDismojiByName = name => {
-  for (let cat of Object.values(dismoji)) {
-    if (typeof cat[name] != 'undefined') {
+const getDismojiByName = (name) => {
+  const namedDismoji = Object.values(dismoji).map((cat) => {
+    if (typeof cat[name] !== 'undefined') {
       return cat[name];
     }
-  }
-  return null;
-}
+    return null;
+  });
+  return namedDismoji;
+};
 
-const getDismojiByUnicode = uni => {
-  for (let cat of Object.values(dismoji)) {
-    for (let emo of Object.values(cat)) {
-      if (emo === uni){
+const getDismojiByUnicode = (uni) => {
+  const uniDismoji = Object.values(dismoji).map((cat) => {
+    const uniDismoji1 = Object.values(cat).map((emo) => {
+      if (emo === uni) {
         return true;
       }
-    }
-  }
-  return false;
-}
-
-const findEmojis = async text => await Promise.all(Array.from(text.matchAll(/(?<=\u200b):?([^:\n]+):?(?=\u200b)/g), getEmoji));
+      return null;
+    });
+    return uniDismoji1;
+  });
+  return uniDismoji;
+};
 
-const replaceEmojis = (text, emojis) => emojis.filter(emoji => emoji.replacement).reduce((curText, emoji) => curText.replace(emoji.text, emoji.replacement), text);
+const findEmojis = async (text) => await Promise.all(Array.from(text.matchAll(/(?<=\u200b):?([^:\n]+):?(?=\u200b)/g), getEmoji));
 
+const replaceEmojis = (text, emojis) => emojis.filter((emoji) => (
+  emoji.replacement)).reduce((curText, emoji) => (
+  curText.replace(emoji.text, emoji.replacement)), text);
 
-const getEmoji = async match => {
+const getEmoji = async (match) => {
   const guild = client.guilds.cache.get(GUILD_ID);
 
-  let text = match[0]
-  let name = match[1];
+  const text = match[0];
+  const name = match[1];
 
-  let emoji = {
-    'text': text,
-    'name': name,
-    'replacement': false,
+  const emoji = {
+    text,
```

Comment:
Huh, I hadn't seen this syntax before. Neat.

Reply:
Yeah, that was just an auto-replace from the linter.